cmd/kubelet/kubelet.go

```diff
@@ -100,10 +100,13 @@ func main() {
 		glog.Info(err)
 	}
 
+	client, err := standalone.GetAPIServerClient(*authPath, apiServerList)
+	if err != nil {
+		glog.Warningf("No API client: %v", err)
```

Comment:
This is not a warning, since people use kubelets without an apiserver on purpose.
If you want to warn, then warn when apiServerList is non-empty, and it can't setup a client.